packages/protocol/contracts/libs/LibInvalidTxList.sol

```diff
@@ -47,7 +47,7 @@ library LibInvalidTxList {
         TX_GAS_LIMIT_TOO_SMALL
     }
 
-    function isTxListInvalid(
+    function assertTxListInvalid(
```

Comment:
if we use `assertTxListInvalid`, the implementation in this method shall also be changed to have actual assessions. For example, 

```solidity
if (encoded.length > config.maxBytesPerTxList) {
    return Reason.BINARY_TOO_LARGE;
}
```
should be changed to:
```solidity
require(hint != OK);

if (hint == Reason.BINARY_TOO_LARGE) {
  require(encoded.length > config.maxBytesPerTxList, "error");
}
```

Reply:
@dantaik I was thinking to change it to custom error in this PR as we are moving that direction.

Reply:
I have a PR for "custom errors", I'll make it ready for review over the weekend.